docs/flakiness/border-radius.mdx

```diff
@@ -0,0 +1,14 @@
+---
+slug: /border-radius
+sidebar_position: 40
+---
+
+# Border Radius
```

Comment:
It is actually a glitch, I don't think you should move it to another section.

Reply:
No one will click on Glitch's page to find a fix for unstable border-radius. Don't you think that it will be more convenient to have a dedicated page for this recipe ?

Reply:
No because we have a search, personally I would click on glitches and not border radius.